rero_ils/theme/views.py

```
@@ -71,6 +71,7 @@ def decorated_view(*args, **kwargs):
 @blueprint.route("/error")
 def error():
     """Error to generate exception for test purposes."""
+    current_app.logger.info("Generating an exception for test purposes.")
     raise Exception("this is an error for test purposes")
```

Ruff (0.15.1)

[warning] Create your own exception

(TRY002)